<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElectricalForm</class>
 <widget class="QWidget" name="ElectricalForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define options for the electric model.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Electric model</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBoxScaling">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Specify whether an electric variables scaling should be done. In that case, the boundary conditions for the potential will be tuned at each time step in order to reach a user-specified target dissipated power (Joule effect) or a user-specified target current intensity (electric arcs).&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(Code_Saturne key word: &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;IELCOR&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Electric variables scaling</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>64</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBoxElectricArc">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Specify electric model options in case of electric arcs.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="title">
         <string>Electric Arc</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the imposed current intensity. This is the target current intensity (A) for the calculations with boundary condition tuning for the potential. The target intensity will be reached if the boundary conditions are expressed using the variable &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;dpot&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; or if the initial boundary conditions are multiplied by the variable &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;coejou&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(Code_Saturne key word: &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;COUIMP&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Imposed current
intensity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditCurrent">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the imposed current intensity. This is the target current intensity (A) for the calculations with boundary condition tuning for the potential. The target intensity will be reached if the boundary conditions are expressed using the variable &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;dpot&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; or if the initial boundary conditions are multiplied by the variable &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;coejou&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(Code_Saturne key word: &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;COUIMP&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelScalingModel">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Select the variables scaling model between:&lt;br/&gt;- General case: the boundary condition tuning is done using the global power.&lt;br/&gt;- Plane define: define a scaling plane for boundary condition tuning. Variables will be multiplied by a certain coefficient so that the integral over the plane surface of the current density is equal to the imposed current intensity.&lt;br/&gt;- User define: define an electric scaling using the user subroutine &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;cs_user_electric_scaling&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(Code_Saturne key word: &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;MODREC&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Variables scaling model</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxScalingModel">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Select the variables scaling model between:&lt;br/&gt;- General case: the boundary condition tuning is done using the global power.&lt;br/&gt;- Plane define: define a scaling plane for boundary condition tuning. Variables will be multiplied by a certain coefficient so that the integral over the plane surface of the current density is equal to the imposed current intensity.&lt;br/&gt;- User define: define an electric scaling using the user subroutine &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;cs_user_electric_scaling&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(Code_Saturne key word: &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;MODREC&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1" rowspan="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>240</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBoxRecalage">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the scaling plane parameters.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="title">
            <string>Scaling plane parameters</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the current density direction.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Current density
direction</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxDirection">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the current density direction.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>167</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labePlaneDefinition">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Plane definition</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>167</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="lineEditPlaneDefinitionA">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>X +</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditPlaneDefinitionB">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Y +</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditPlaneDefinitionC">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Z +</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditPlaneDefinitionD">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define the plane equation coefficients &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; where &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;aX + bY + cZ = 0&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>= 0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Epsilon</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditEpsilon"/>
              </item>
             </layout>
            </item>
            <item row="3" column="1">
             <spacer name="horizontalSpacer_13">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>182</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBoxJoule">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Specify electric model options in case of Joule model.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="title">
         <string>Joule model</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" rowspan="2">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>56</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBoxJouleModel">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Select the Joule model between:&lt;br/&gt;- AC/DC,&lt;br/&gt;- Three-phase.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Specify the imposed power in case of a Joule model. This is the target dissipated power (W) for the calculations with boundary condition tuning for the potential.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;The target power will be reached if the boundary conditions are expressed using the variable &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;dpot&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; or if the initial boundary conditions are multiplied by the variable &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;coejou&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(Code_Saturne key word: &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;PUISIM&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Imposed power</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditPower">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Specify the imposed power in case of a Joule model. This is the target dissipated power (W) for the calculations with boundary condition tuning for the potential.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;The target power will be reached if the boundary conditions are expressed using the variable &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;dpot&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; or if the initial boundary conditions are multiplied by the variable &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;coejou&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(Code_Saturne key word: &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;PUISIM&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources_pages.qrc"/>
 </resources>
 <connections/>
</ui>
